flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1270,11 +1445,6 @@ class BlockDeviceCalculatorTests(TestCase):
     """
     Tests for ``BlockDeviceCalculator``.
     """
-    def setUp(self):
-        super(BlockDeviceCalculatorTests, self).setUp()
-        self.deployer = create_blockdevicedeployer(self)
-        self.persistent_state = InMemoryStatePersister()
```

Comment:
Probably. Not going to do this here. There is probably a better abstraction we could use to make this cleaner.